src/inmanta_module_factory/helpers/utils.py

```diff
@@ -24,12 +24,247 @@
 import tempfile
 
 import inmanta
+import inmanta.ast.type
 import inmanta.module
+import inmanta.parser.plyInmantaLex
 
 from inmanta_module_factory.helpers import const
 
 LOGGER = logging.getLogger(__name__)
 
+assert inmanta.parser.plyInmantaLex.t_ID.__doc__ is not None, "Invalid lexer specifications"
```

Comment:
I'm hesitant to make this part of the stable API because it means we lose the freedom to change our lexer/parser framework. But I take it that's acceptable for this package?

Reply:
Indeed, this is purely offline code